lib/src/chat_view.dart

```diff
@@ -527,79 +527,87 @@ class DashChatState extends State<DashChat> {
-                        trailing: widget.trailing,
-                        inputContainerStyle: widget.inputContainerStyle,
-                        inputTextStyle: widget.inputTextStyle,
-                        inputFooterBuilder: widget.inputFooterBuilder,
-                        inputCursorColor: widget.inputCursorColor,
-                        inputCursorWidth: widget.inputCursorWidth,
-                        alwaysShowSend: widget.alwaysShowSend,
-                        scrollController: widget.scrollController != null
-                            ? widget.scrollController
-                            : scrollController,
-                        focusNode: inputFocusNode,
-                        reverse: widget.inverted,
-                      ),
-                    )
+                  Column(
+                    children: [
+                      if (widget.messages.length != 0 &&
+                          widget.messages.last.user.uid != widget.user.uid &&
+                          widget.messages.last.quickReplies != null)
+                        Container(
+                          padding: widget.quickReplyPadding,
+                          constraints: BoxConstraints(
+                              maxHeight:
+                                  widget.quickReplyScroll ? 50.0 : 100.0),
+                          width: widget.quickReplyScroll ? null : maxWidth,
+                          child: widget.quickReplyScroll
+                              ? ListView(
+                                  scrollDirection: Axis.horizontal,
+                                  children: widget
+                                      .messages.last.quickReplies.values
+                                      .map(_mapReply)
+                                      .toList(),
+                                )
+                              : Wrap(
+                                  children: <Widget>[
+                                    ...widget.messages.last.quickReplies.values
+                                        .sublist(
+                                            0,
+                                            widget.messages.last.quickReplies
+                                                        .values.length <=
+                                                    3
+                                                ? widget.messages.last
+                                                    .quickReplies.values.length
+                                                : 3)
+                                        .map(_mapReply)
+                                        .toList(),
+                                  ],
+                                ),
+                        ),
+                      if (widget.chatFooterBuilder != null)
+                        widget.chatFooterBuilder(),
+                      if (!widget.readOnly)
+                        SafeArea(
+                          top: false,
```

Comment:
turning the top off for the input helps remove some unnecessary space between the input and the chat items